.changeset/honest-olives-kneel.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Icons added: accessibility, channel, minus-circle-fill, plus-circle-fill  
+Icons changed: plus-circle
```

Comment:
Updated to align with the [content guidelines](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#new-icons) for changelogs.
```suggestion
`accessibility`, `channel`, `minus-circle-fill`, `plus-circle-fill` icons added.
Fixed the size and position of the `plus-circle` icon.
```